src/common/config.ts

```diff
@@ -18,11 +18,27 @@ function loadExcludedNamespaces(): string[] | null {
   }
 }
 
+function getClusterName(): string {
+  if (!config.CLUSTER_NAME) {
+    return 'Default cluster';
+  }
+
+  if (config.CLUSTER_NAME.includes('/')) {
+    // logger is not yet created so defaulting to console.log
+    console.log(
+      `clusterName (${config.CLUSTER_NAME}) contains disallowed character "/". Replacing with ":".`,
```

Comment:
Why you have chosen `:` actually? 😃

Reply:
had to chose something 🤷  maybe it should be space? what do you want it to be? also I know `:` works

Reply:
just removing it now 👍

Reply:
The log will still say that you're replacing with `:`

Reply:
oh yeah, forgot about that 😆

Reply:
fixed